### PR TITLE
Fix up Identifier

### PR DESCRIPTION
**Pull request by natanfudge**

### CODE REVIEW

File: mappings/net/minecraft/util/Identifier.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
+	COMMENT The namespace and path must only contain either [a-z] (a letter), [0-9] (a digit), or the characters '_', '.', and '-'.
```
Comment (Pyrofab) [inline mappings/net/minecraft/util/Identifier.mapping]: Doesn't the path accept `/` too ?

File: mappings/net/minecraft/util/Identifier.mapping [hunk 2]
```diff
@@ -28,4 +29,12 @@ CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
 		ARG 0 path
 	METHOD method_20209 isNamespaceValid (Ljava/lang/String;)Z
 		ARG 0 namespace
+	METHOD <init> (Ljava/lang/String;Ljava/lang/String;)V
+		ARG 1 namespace
+		ARG 2 path
+	METHOD <init> (Ljava/lang/String;)V
+		ARG 1 id
+			COMMENT A string of the form <path>:<namespace>, for example minecraft:iron_ingot.
```
Comment (liach) [inline mappings/net/minecraft/util/Identifier.mapping]: namespace:path not path:namespace :angry:

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/util/Identifier.mapping [hunk 3]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
+	COMMENT The namespace and path must only contain either [a-z] (a letter), [0-9] (a digit), or the characters '_', '.', and '-'. The path can also have a '/'.
```
Comment (Pyrofab) [inline mappings/net/minecraft/util/Identifier.mapping]:
```suggestion
	COMMENT The namespace and path must contain only lowercase letters ([a-z]), digits ([0-9]), or the characters '_', '.', and '-'. The path can also contain the standard path separator '/'.
```
I think this is preferable to a less verbose RegEx description for doc.

File: mappings/net/minecraft/util/Identifier.mapping (hunk 2 above)
Comment (natanfudge) [inline mappings/net/minecraft/util/Identifier.mapping]:
```suggestion
			COMMENT A string of the form <namespace>:<path>, for example minecraft:iron_ingot.
```

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by Pyrofab — APPROVED** [review]

**Pull request merged**